src/graphics/opengl.c

```diff
@@ -135,6 +135,7 @@ struct Canvas {
   int attachmentCount;
   bool needsAttach;
   bool needsResolve;
+  bool immortal;
```

Comment:
This is meant to communicate "If true, framebuffer resource lifetimes are externally managed, if false, framebuffer resources are managed by Lovr and Lovr will delete framebuffers when the object is deleted".